### PR TITLE
Sanity Updates - 2023-10-10

### PR DESCRIPTION
**Pull request by wrp801**
Updated Blueprint Library files with latest data from Sanity. 

 Run for *[_type == "vendors" && name match "Domo" && !(_id in path("drafts.**"))]

### CODE REVIEW

File: docs/blueprint-library/domo/domo-overview.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Shipyard currently has the following Blueprints readily available:
 - [Upload File as Dataset from Shipyard](domo-upload-csv-as-dataset.md)
 - [Download Card to Shipyard](domo-download-card.md)
 - [Download Dataset to Shipyard](domo-download-dataset.md)
-- [Refresh Dataset](domo-refresh-dataset.md)
+- [Trigger Dataset Refresh](domo-refresh-dataset.md)
 
 ## Open Source Code
-The code that powers these Blueprints is [available on our Github](https://github.com/shipyardapp/domo-blueprints), open sourced under the Apache License 2.0. We'd love to have you contribute to others using Shipyard Blueprints by creating issues or submitting PRs.
+The code that powers these Blueprints is [available on our Github](http://localhost:3000/docs/blueprint-library/domo/domo-overview/), open sourced under the Apache License 2.0. We'd love to have you contribute to others using Shipyard Blueprints by creating issues or submitting PRs.
```
Comment (johnathan-rodriguez) [inline docs/blueprint-library/domo/domo-overview.md:32]: fix github link

File: docs/blueprint-library/domo/domo-upload-csv-as-dataset.md [hunk 2]
```diff
@@ -28,39 +28,40 @@ Create a Domo dataset by uploading a csv to Domo. If the `Domo Dataset Name` alr
...
-    - "200"
-    - "201"
-    - "203"
-    - "204"
-    - "205"
-    - "210"
-    - "211"
-    - "214"
-    - "215"
-    - "216-220"
```
Comment (johnathan-rodriguez) [inline docs/blueprint-library/domo/domo-upload-csv-as-dataset.md:65]: no action needed but worth highlight exit code 220 has been removed from the guardrails which was `EXIT_CODE_SYNC_CHECK_ERROR`

just confirming this was intentional

**Review by johnathan-rodriguez — COMMENTED** [review]
one small change around the github link

File: docs/blueprint-library/domo/domo-overview.md (hunk 1 above)
Comment (wrp801) [reply docs/blueprint-library/domo/domo-overview.md]: this is fixed [](edited-with-Axolo)

File: docs/blueprint-library/domo/domo-upload-csv-as-dataset.md (hunk 2 above)
Comment (wrp801) [reply docs/blueprint-library/domo/domo-upload-csv-as-dataset.md:65]: yes this is intentional [](edited-with-Axolo)

**Pull request merged**